views/templates/admin/configuration.tpl

```diff
@@ -55,7 +55,7 @@
 		{l s='Errors list:' mod='mailjet'}
 		<ul class="mj_errors">
 			{foreach from=$MJ_errors item=current_error}
-				<li>{$current_error|escape|default:''}</li>
+				<li>{$current_error|default:''}</li>
```

Comment:
Rollback from security review ? Not really good ... If this is because of HTML code, have you tried another escape option ? Like `{...|escape:'javascript'}` ?